src/compas_view2/app/app.py

```diff
@@ -273,3 +280,15 @@ def _add_action(self, item, parent):
             icon = self._get_icon(item['icon'])
             return parent.addAction(icon, text, partial(action, *args, **kwargs))
         return parent.addAction(text, partial(action, *args, **kwargs))
+
+    def animate(self, interval=100):
```

Comment:
it would also be a good thing if we follow a certain naming convention for callback decorators. something like `on_...` to signal that the trigger is base don some kind of event?